.vsts-ci/azure-pipelines-ci.yml

```diff
@@ -9,26 +9,36 @@ variables:
     value: 'true'
 
 trigger:
-  batch: true
   branches:
     include:
       - master
 
+resources:
+  repositories:
+  - repository: PowerShellEditorServices
+    type: github
+    endpoint: GitHub
+    name: PowerShell/PowerShellEditorServices
+    ref: master
+
 jobs:
-- job: Windows
+- job: Win2019
+  displayName: Windows Server 2019
```

Comment:
Is there a reason we're on 2019 here and 2016 in PSES?

Reply:
PSES has both 2016 and 2019.